CRM/Event/Form/Registration/Confirm.php

```diff
@@ -662,7 +662,6 @@ public function postProcess() {
         $allParticipantIds = array_merge([$registerByID], $this->_additionalParticipantIds);
       }
 
-      $entityTable = 'civicrm_participant';
```

Comment:
The changes in this file seem like a no-brainer - if you felt like pulling them out I could quick-merge them

Reply:
Thanks, merged via https://github.com/civicrm/civicrm-core/pull/32225